mdx-layout-tailwind/src/styles.css

```diff
@@ -0,0 +1,7 @@
+article {
+  padding: 1rem;
+}
+
+nav {
+  padding: 1rem;
```

Comment:
nav should be also parametrized with components ... and adding `p-4` inside className
same for article ... you should add `p-4` in className of a component